assembly/runtime/util.ts

```diff
@@ -68,7 +73,9 @@ export namespace util {
   }
 
   /**
+   * @package
```

Comment:
cool, when do we use this?

Reply:
This is a jsdoc thing to indicate that this should be only used internally in the package. I haven't found other usages though.